README.md

```diff
@@ -1,94 +1,74 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/cubzh/cubzh/main/misc/logo_and_name_light.svg">
-  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/cubzh/cubzh/main/misc/logo_and_name_dark.svg">
-  <img alt="" src="https://raw.githubusercontent.com/cubzh/cubzh/main/misc/logo_and_name_dark.svg">
+  <source media="(prefers-color-scheme: dark)" srcset="misc/logo_and_name_light.svg">
+  <source media="(prefers-color-scheme: light)" srcset="misc/logo_and_name_dark.svg">
+<p align="center">
+  <img width=50% alt="" src="misc/logo_and_name_dark.svg">
+</p>
 </picture>
 
 ![CI](https://github.com/cubzh/cubzh/actions/workflows/ci.yml/badge.svg?branch=main)
 [![Join the chat at https://cu.bzh/discord](https://img.shields.io/discord/355905150528913409?color=%237289DA&label=cubzh&logo=discord&logoColor=white)](https://cu.bzh/discord)
 
-Cubzh is a voxel game platform. (*voxel* == *volumetric pixel* == *cube*, in that context)
+## What is Cubzh?
 
-It all started as a [Lua](https://www.lua.org/) scripting environment focused on voxel manipulation & real-time communication.
+Cubzh is an online platform written in C/C++ and Lua, allowing users to create Items, Environments, and Games using cubes paired with Lua scripts. It is inpired by both Roblox and Minecraft. With an easy-to-use scripting environment and free servers for instant multiplayer action, we're aiming to unleash the kind of creativity you see in Roblox. Enforcing cubes as building blocks for all 3D assets makes modeling and collaboration easier.
 
-Our goal is to build the most exciting and accessible place for artists, developers & players.
+All features are bundled into one comprehensive cross-platform application.  eliminating the need for a separate "studio" app for developers. We would like all users to get a chance to bump into each other, wether they're here as wanderers, players, artists or coders. 
```

Comment:
`application.  eliminating`

- should there be a capital E ?
- also there are 2 spaces after the `.`